.travis.yml

```diff
@@ -11,6 +11,7 @@ matrix:
   include:
     - php: 7.1
       env:
+        - RELEASE=1
```

Comment:
This entry will always default to using SYMFONY 4.0.x.

I'd rather do the release on the last matrix entry that uses the `composer.lock` file as committed.

Reply:
Very good point

Reply:
fixed